lib/withOnyx.js

```diff
@@ -87,20 +87,20 @@ export default function (mapOnyxToState) {
              * @param {*} val
              */
             setInitialState(statePropertyName, val) {
-                if (!this.state.loading) {
```

Comment:
Mostly out of curiosity: why are we reversing this `if`?

Reply:
The logic has changed from what we had before which was incorrect.

Reply:
We don't need to invert the logic though it's just how I did it. Updated so this is unchanged but the logic is the same.